community/modules/scheduler/schedmd-slurm-gcp-v5-controller/outputs.tf

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-output "controller_instance_id" {
+output "controller_instance_ids" {
   description = "The server-assigned unique identifier of the controller compute instance."
-  value       = module.slurm_controller_instance.slurm_controller_instance.instances_details[0].id
+  value       = module.slurm_controller_instance.slurm_controller_instance.instances_details[*].id
```

Comment:
I would suggest using `one` here instead of changing to a list. I think it would avoid the failure and pass null instead of breaking on a lookup. 

https://developer.hashicorp.com/terraform/language/functions/one

Reply:
Good suggestion, it's a lot cleaner this way. 
Fixed